src/storage/entry_conversion.py

```diff
@@ -142,17 +146,23 @@ def _sanitize_value(analysis_data: dict, key: str, value):
 
 
 def _sanitize_string(string: str) -> str:
-    string = string.replace('\0', '')
     try:
         string.encode()
     except UnicodeEncodeError:
         string = string.encode(errors='replace').decode()
+    # replace all characters that are converted to unicode characters in JSON, because unicode characters can't be
+    # saved in the PostgreSQL database
+    json_string = json.dumps(string)
+    if JSON_UNICODE_REGEX.search(json_string):
+        logging.warning(f'Sanitizing unicode characters in string {json_string[100:]}')
```

Comment:
Why the `[100:]` here?

Reply:
The string could be very long (there are some big analysis results which are stored as strings) and I thought not printing a few MBs worth of string would generally be a good idea